internal/authorization/converters.go

```diff
@@ -55,15 +62,20 @@ func (c IdentityConverter) TypeName() string {
 
 func (c IdentityConverter) Map(r *http.Request) []Permission {
 	id := chi.URLParam(r, "id")
+	var contextualTuples []openfga.Tuple
 
 	if id == "" {
-		// global is used in place of `*`, reason is OpenFGA has a special meaning for `*`
-		// see [public access](https://openfga.dev/docs/modeling/public-access)
-		id = "global"
+		id = GLOBAL_ACCESS_OBJECT_NAME
+		// Add contextual tuples to enable all users to list resources and give admins access to all resources
+		contextualTuples = append(
+			contextualTuples,
+			*openfga.NewTuple("user:*", CAN_VIEW, fmt.Sprintf("%s:%s", c.TypeName(), GLOBAL_ACCESS_OBJECT_NAME)),
+			*openfga.NewTuple("privileged:superuser", ADMIN_OBJECT, fmt.Sprintf("%s:%s", c.TypeName(), GLOBAL_ACCESS_OBJECT_NAME)),
```

Comment:
issue: ADMIN_OBJECT is equal to `privileged:user` (which identifies a type `privileged` with identifier `user`) which is not the name of an existing relation in the auth model. This should not work as it is.
I think it is supposed to be:


```go
*openfga.NewTuple(
    "privileged:superuser",
    "privileged",
    fmt.Sprintf("%s:%s", c.TypeName(), GLOBAL_ACCESS_OBJECT_NAME)
),
``` 

where `privileged` is the name of the relation of the type `identity` for example. Same goes for other types

I'll put a request changes for this just to alert, please double check

Reply:
in all fairness this is not needed at all, the link between the `privileged` superuser and the objects is done at creation,  so only one that needs to be provided is the `user:*`

i'll try to set this up via openfga diagram to explain better 

```suggestion
```

Reply:
my mistake, `privileged` contextual tuple needs to be there for other action types (like create or edit) and allows users in provided they are superusers